src/runtime/offers-flow-test.js

```diff
@@ -155,6 +159,7 @@ describes.realWin('OffersFlow', {}, env => {
           showNative: true,
           list: 'default',
           skus: null,
+          isClosable: undefined,
```

Comment:
I'd rather default explicitly in the over-the-boundary requests. This way the default only rests in one place.